/**:
  ros__parameters:
    backpropagation:
      gain_function_u: [-1., -0.5, -0.05, 0., 0.05, 0.5, 1.]
      gain_function_positive_y: [0., 0., 0., 0.5, 0.9, 1., 1.]
      gain_function_negative_y: [1., 1., 0.9, 0.5, 0., 0., 0.]
